### ISSUE TITLE
Feature request: Add flag to status to include nested CondaPkg.toml definitions

### ISSUE THREAD

**Issue by schlichtanders** [JuliaPy/CondaPkg.jl#147]
Hi there, it would be nice to have an easy way of listing all active dependencies and from which nested CondaPkg.toml they come from.
A flag to CondaPkg.status also mentioned in the docstring would be ideal.

**Comment by cjdoris** [comment]
Agreed this would be nice. In the meantime you can do `pkg> conda run conda list` to list all installed packages.
